styles.css

```diff
@@ -537,15 +537,15 @@ hr {
   position: fixed;
   top: 0;
   left: 0;
-  background-color: white;
+  background-color: #6070ff;

```

Comment:
- [ ] Good job so far 👍 but you'll notice that in the Figam template, the background is semi-transparent and as a result, you're able to see a blurred version of the page underneath it. It would be great to add this functionality so as to follow the template.
- [ ] Also the links in the mobile menu are supposed to be aligned to the left but yours are somehow aligned to the center. Kindly correct this.
- Your design:
![Annotation 2022-03-08 130331](https://user-images.githubusercontent.com/85113567/157235171-4d7016fc-4c49-4661-b36b-38fe1fff4791.jpg)
- Figma design:
![Annotation 2022-03-08 100502](https://user-images.githubusercontent.com/85113567/157235208-b280106f-b3cd-44ea-b66b-526de3cfe143.jpg)

Reply:
Done

Reply:
Done